trio/_core/_tests/test_run.py

```diff
@@ -621,12 +619,11 @@ async def test_basic_timeout(mock_clock: _core.MockClock) -> None:
 
 async def test_cancel_scope_nesting() -> None:
     # Nested scopes: if two triggering at once, the outer one wins
-    with _core.CancelScope() as scope1:
-        with _core.CancelScope() as scope2:
-            with _core.CancelScope() as scope3:
-                scope3.cancel()
-                scope2.cancel()
-                await sleep_forever()
+    with _core.CancelScope() as scope1, _core.CancelScope() as scope2:  # noqa: SIM117  # multiple-with-statements
+        with _core.CancelScope() as scope3:
```

Comment:
Why combine 2 and leave one behind?

Reply:
Disabled combining with statements in 76d7413